src/dispatch_common.c

```diff
@@ -670,13 +670,23 @@ epoxy_load_gl(void)
     get_dlopen_handle(&api.gl_handle, OPENGL_LIB, true, true);
 #else
 
+    // Prefer GLX_LIB over OPENGL_LIB to maintain existing behavior.
+    // Without this ordering, dependent projects experienced bugs such as:
```

Comment:
Tiniest of nits:

```suggestion
    // Using the inverse ordering OPENGL_LIB -> GLX_LIB, causes issues such as:
```

With that the MR is:
```
Reviewed-by: Emil Velikov <emil.velikov@collabora.com>
```